ArmRobot/Assets/Scripts/DomainRandomization.cs

```diff
@@ -0,0 +1,218 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DomainRandomization : MonoBehaviour
+{
+    public GameObject cube;
+    public GameObject robot;
+    public GameObject table;
+
+
+    public float scaleObject = 0.1f;
+
+    public float robotMinReach = 0.2f;
+    public float robotMaxReach = 0.5f;
+
+    public int nbMaxCylinders = 5;
+    public int nbMaxSpheres = 5;
+    public float rotationAngle = 10.0f;
+
+    public float yAltitudeTable = 0.813f;
+    public float minimumDistanceWithoutObjects = 0.05f;
+
+    public int nbMaxLights = 3;
+
+    
+    void start()
+    {
+        // first we set the position of the cube 
+        float minimumDistanceBetweenObjects = scaleObject + minimumDistanceWithoutObjects;
+
+        cube.transform.position = new Vector3(-robotMaxReach + 2*minimumDistanceWithoutObjects, 
```

Comment:
Why do we need to set the position of the cube, non-randomly, here? I assume its being set randomly elsewhere?

Reply:
It is only the initial position

Reply:
Why do we need to set a new initial position for the cube at all, if we are just going to move it again before each datapoint is collected anyway? It already has the position it was in in the scene, right?

Reply:
yes I removed this part